tests/bugs/client/issue-2337-lock-mem-leak.t

```diff
@@ -1,42 +0,0 @@
-#!/bin/bash
```

Comment:
We should be removing `tests/bugs/client/issue-2337-lock-mem-leak.c` as well.

Reply:
Done